proxy-node-acceptance-tests/Gemfile.lock

```diff
@@ -0,0 +1,81 @@
+GEM
```

Comment:
This probably can be gitignored, if we need it to be statically version we can just add the version to the Gemfile

Reply:
I think checking in the `Gemfile.lock` is fine, since we're not pinning our gem versions.